src/doom/m_menu.c

```diff
@@ -1045,15 +1045,21 @@ void M_QuickSaveResponse(int key)
     if (key == key_menu_confirm)
     {
 	M_DoSave(quickSaveSlot);
+	if (!S_StartSoundOptional(NULL, sfx_mnucls)) // [NS] Optional menu sounds.
```

Comment:
I'd prefer if you could condense most of this into *one* new function that takes two sound ids as arguments, the prefered one and the fallback/original one.

Reply:
I debated doing this originally so it would be one-line changes. I can set it up so NULL for the second argument plays nothing.

Reply:
Okay, show me. 😁